common/static/common/js/common_libraries.js

```diff
@@ -1,22 +0,0 @@
-define([
-    'edx-ui-toolkit/js/utils/string-utils',
-    'edx-ui-toolkit/js/utils/html-utils',
-    'domReady!',
-    'jquery',
-    'jquery-migrate',
-    'backbone',
-    'underscore',
-    'gettext'
-],
-    function(StringUtils, HtmlUtils) {
-        'use strict';
-
-        // Install utility classes in the edX namespace to make them
-        // available to code that doesn't use RequireJS,
-        // e.g. XModules and XBlocks.
-        if (window) {
-            window.edx = window.edx || {};
-            window.edx.StringUtils = StringUtils;
```

Comment:
Do StringUtils and HtmlUtils still end up in the edx namespace (or are the directly on the window object)?